src/Parser/Hunter/BeastMastery/Modules/Talents/BestialFury.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import getDamageBonus from "Parser/Hunter/Shared/Core/getDamageBonus";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import ITEMS from "common/ITEMS/HUNTER";
+
+//Calculates the actual % increase in damage from Bestial Fury
+const BESTIAL_FURY_MODIFIER = 1 - (1 / 1.4 * 1.25);
```

Comment:
I think this calculation is wrong, this comes out to ~0.107, while I think the correct calculation is `(1.4 / 1.25) - 1`, which is 0.12.

I'll use the following terminology: a "multiplier" is the amount you multiply with damage to get new damage, e.g. 1.4, and a "bonus" is the amount of the multiplier over 100%, which is multiplier - 1, e.g. 0.4. The correct formula is `newMultiplier / oldMultiplier === relativeMultiplier === relativeBonus + 1`

If you want a more concrete example why my formula is correct: if X is the old multiplier and Y is the new multiplier, your formula is `1 - (1 / Y * X)`, mine is `(Y / X) - 1`. Lets consider X = 1.0 and Y = 1.5. I think we easily see without a formula that this is a relative increase of 50%, since "X = 1.0" means we started with no bonus at all. Your formula does `1 - (1 / 1.5 * 1) === 0.33...` mine does `(1.5 / 1) - 1 === 0.5`.